### PR TITLE
chore: lower pre-commit timeout in E2E tests

### PR DESCRIPTION
**Pull request by alexanderbez**
Lower's the `precommit-timeout` to 5ms in the e2e tests.

ref: #2100

cc @ValarDragon @p0mvn

### CODE REVIEW

**Comment by alexanderbez** [comment]
Looks like this didn't help @ValarDragon, unless I changed the wrong config value?

**Comment by czarcas7ic** [comment]
@alexanderbez there are also hard coded wait times in order to allow all Validators to vote on and gov prop. Your change should have still at least reduced the time to wait between txs though..

**Comment by ValarDragon** [comment]
This was the wrong param, should've been precommit!

File: tests/e2e/initialization/node.go [hunk 1]
```diff
@@ -310,6 +311,7 @@ func (n *internalNode) initValidatorConfigs(c *internalChain, persistentPeers []
 	valConfig.StateSync.Enable = false
 	valConfig.LogLevel = "info"
 	valConfig.P2P.PersistentPeers = strings.Join(persistentPeers, ",")
+	valConfig.Consensus.TimeoutCommit = 5 * time.Millisecond
```
Comment (p0mvn) [inline tests/e2e/initialization/node.go:314]: Note that for this change to take effect we need to backport this to `v10.x` and rebuild the chain-init image. Then, it has to be updated here:
https://github.com/osmosis-labs/osmosis/blob/8e6e4de0bc65d521185f2aa82ff5884c107bd8c5/tests/e2e/containers/config.go#L30

If the `initialization` package is updated, the `previousVersionInit` tag should be updated as well

This is a manual step to make sure that we are able to initialize genesis and configs of the previous Osmosis version correctly.

**Comment by ValarDragon** [comment]
Closing in favor of #2103 

(I switched the config to TM's test Config inspired by your change, and then found source of a lot of the delay via logging)

**Comment by alexanderbez** [comment]
Man, this e2e stuff has really evolved from what I had originally designed 😆

**Pull request closed without merging**